collector/fixtures/e2e-output.txt

```diff
@@ -2289,6 +2289,58 @@ node_nfsd_server_rpcs_total 18628
 # HELP node_nfsd_server_threads Total number of NFSd kernel threads that are running.
 # TYPE node_nfsd_server_threads gauge
 node_nfsd_server_threads 8
+# HELP node_power_supply_capacity capacity value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_capacity gauge
+node_power_supply_capacity{power_supply="BAT0"} 81
+# HELP node_power_supply_capicity_level capicity_level value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_capicity_level gauge
+node_power_supply_capicity_level{capacity_level="Normal",power_supply="BAT0"} 1
+# HELP node_power_supply_cyclecount cyclecount value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_cyclecount gauge
+node_power_supply_cyclecount{power_supply="BAT0"} 0
+# HELP node_power_supply_energy_full energy_full value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_energy_full gauge
+node_power_supply_energy_full{power_supply="BAT0"} 4.507e+07
+# HELP node_power_supply_energy_full_design energy_full_design value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_energy_full_design gauge
+node_power_supply_energy_full_design{power_supply="BAT0"} 4.752e+07
+# HELP node_power_supply_energy_now energy_now value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_energy_now gauge
+node_power_supply_energy_now{power_supply="BAT0"} 3.658e+07
+# HELP node_power_supply_manufacturer manufacturer value of /sys/class/power_supply/<power_supply>.
+# TYPE node_power_supply_manufacturer gauge
+node_power_supply_manufacturer{manufacturer="LGC",power_supply="BAT0"} 1
```

Comment:
I think all these 'info' metrics should be one metric like:

```
node_power_supply_info{manufacturer, model_name, serial_number, status, technology}
```

Reply:
I combined all string informations to a single `node_power_supply_info` metric